config/config.json

```diff
@@ -61,9 +61,9 @@
         },
         "kafka": {},
         "masSso": {
```

Comment:
I think we should remove masSso object as we don't need it anymore.

Reply:
I agree and I tried initially, but it failed to loading here. I think it first needs to be removed from the `Config` in `@rhoas/app-services-ui-shared`.
https://github.com/redhat-developer/app-services-ui/blob/642cfdfa44f60fc129fc15a411fb376274104505/src/app/providers/config/EmbeddedConfigContextProvider.tsx#L15-L16